C/manual/tips-appendix.docbook

```diff
@@ -6457,4 +6316,662 @@ bogus_symbol2,price2,date2
       <ulink url="https://rt.cpan.org/Ticket/Attachment/1121440/589997/Tiaacref.pm.zip"/>
     </para>
   </sect1>
+  <sect1 id="fq-deprecated-sources">
+    <title>Deprecated Finance::Quote Sources</title>
+
```

Comment:
Deprecated identifies something that still works but will be eventually removed with eventually meaning some time between the next release and the heat death of the universe.

Obsolete would be better.

Reply:
Oh yes, please.

Reply:
And IMHO the structure should be:
```
<sect1…><title>Finance::Quote Sources</title>…
  <sect2 …> <title>Quote Sources - Individual Sources</title>
    <sect3 …> <title>Active Individual Sources</title>
    <sect3 …><title>Obsolete Individual Sources</title>
  <sect2 …><title>Quote Sources - Multiple Sources</title>
    <sect3 …><title>Active Multiple Sources</title>
    <sect3 …><title>Obsolete Multiple Sources</title>
<sect1…><title>Details of Some Sources</title>…
 <sect2…><title>Yahoo Specifics</title>…
 <sect2…><title>TIAA-CREF Specifics</title>…
```
Less  scrolling for the users searching their old source.

Reply:
> [...] Obsolete would be better.

Updated. Added a bit of abstract too.

> And IMHO the structure should be: [...]

Updated.  It's nicer to my brain indeed. Thanks!